pkg/apis/minio.min.io/v2/helper.go

```diff
@@ -719,16 +719,16 @@ func (t *Tenant) CreateUsers(madmClnt *madmin.AdminClient, userCredentialSecrets
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*20)
 	defer cancel()
 	for _, secret := range userCredentialSecrets {
-		consoleAccessKey, ok := secret.Data["CONSOLE_ACCESS_KEY"]
+		consoleAccessKey, ok := secret.Data["MINIO_ACCESS_KEY"]
```

Comment:
if we are doing this change, perhaps we can drop the `MINIO_` prefix for these two two avoid it getting confused with the root credentials? what do you think @harshavardhana

Reply:
Yes we should drop it and not use the older MINIO_ instead it should be generic like 

ACCESS_KEY